hedera-node/hedera-mono-service/src/main/java/com/hedera/services/contracts/sources/TxnAwareEvmSigsVerifier.java

```diff
@@ -208,6 +209,10 @@ private boolean isActiveInFrame(
             final boolean isDelegateCall,
             final Address activeContract,
             final ContractAliases aliases) {
+        if (key instanceof JKeyList keyList && keyList.isEmpty()) {
```

Comment:
Perhaps this concept - "is sentinel for immutability" is interesting enough it should be pulled out and given a name.  Even though only one line!  Plus the matching function of "create sentinel key for immutability".  (And then make sure they're used wherever the concept is used.) (Plus it might not belong here - might be more appropriate elsewhere - e.g. in `JKeyList` - but I don't know because I know nothing of the structures/situations in which this concept is used (they might all be local here).)